ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -209,13 +214,19 @@ const SV_GENOTYPE_DETAILS = [
   },
 ]
 
+const Comment = styled.p`
```

Comment:
Its really best to use a semantic ui component or existing styled component where possible as it ensures that there will be consistent styles throughout the application. Styled components should ideally only be used when complicated style requirements leave us no other options. In this case, I think using the `subheader` prop of small or tiny `Header` would be best, or if we wanted to keep it really simple you could use a `ColoredDiv` from the `StyledComponents` helpers. The ticket doesn't specify it has to be indented so I think the added complexity isn't really worth it

Reply:
Okay